app/assets/stylesheets/components/_onboarding_message.scss

```diff
@@ -0,0 +1,5 @@
+.onboarding-message {
+  text-align: center;
+  margin: 0;
+  padding: $baseline-unit*6 0;
+}
```

Comment:
The light green box comes from here — https://www.moneyadviceservice.org.uk/en/styleguide/components/common#section-call-out — is there a better way than just mixing these together in a class attr?

It is indeed separate to the flash messages